src/Hit/Cli.hs

```diff
@@ -32,7 +32,7 @@ hit = execParser cliParser >>= \case
     Fix message -> runFix message
     Amend -> runAmend
     Resolve branchName -> runResolve branchName
-    Push isForce -> runPush isForce
+    Push isForce assignOwner -> runPush isForce assignOwner
```

Comment:
I was thinking to add this to the `new` command, so people see that you *started* working on the issue rather than when you *finished*. Because when you `push` most likely that you already did everything in the issue, but the idea in adding assignee is to indicate the issue as occupied.

Reply:
Yes, that is likely the correct command to change. I have made that change.